src/improvements/tasks/TaskRunner.sol

```diff
@@ -91,17 +193,37 @@ contract TaskRunner is Script {
                     Strings.toHexString(uint256(uint160(config.parentMultisig)), 20)
                 )
             );
-            task.signFromChildMultisig(config.path, owners[0]);
+            (accesses,) = task.signFromChildMultisig(config.configPath, owners[0]);
         } else {
-            task.simulateRun(config.path);
+            (accesses,) = task.simulateRun(config.configPath);
         }
     }
 
+    /// @notice Sets the TENDERLY_GAS environment variable for the task if it exists.
+    function setTenderlyGasEnv(string memory basePath) public {
```

Comment:
We want to make sure the tenderly link works for stacked simulations. Therefore we must set the env var before simulating.